docs/harness-attributes.md

```diff
@@ -52,17 +52,18 @@ This allows you to define various environment variables that are set in the dock
 Here you can set a bunch of arbitrary steps to execute before/after go modules have been downloaded. These are executed as you define them, from a shell context, so you can execute other shell scripts from here too. If you want to add a shell script and execute it from these before/after step attributes you can follow these steps:
 
 1. Configure an overlay directory as described in step 1 of [overriding harness templates]
-1. Inside your overlay directory (e.g. `tools/workspace`), add your shell script as `docker/image/app/root/lib/my-script.sh`, the full path will be `tools/workspace/docker/image/app/root/lib/my-script.sh`.
+1. Inside your overlay directory (e.g. `tools/workspace`), add your shell script as `docker/image/app/root/lib/scripts/my-script.sh`, the full path will be `tools/workspace/docker/image/app/root/lib/scripts/my-script.sh`.
+   >_NOTE: The contents of the `scripts` directory will be granted executable permission when the image is being built_
 1. Populate your new shell script with whatever it is you want to execute
-1. Run `ws harness prepare`, and verify that your file has been copied to `.my127ws/docker/image/app/root/lib/` (this directory and its contents are automatically copied onto the Docker image)
+1. Run `ws harness prepare`, and verify that your file has been copied to `.my127ws/docker/image/app/root/lib/scripts` (this directory and its contents are automatically copied onto the Docker image)
```

Comment:
to me it would seem easier to use a project folder's scripts e.g. `/app/tools/scripts` without worrying about an overlay

Reply:
unless of course they want to use the `helper <lib/task command>` stuff

Reply:
yeah, I discussed the idea of leveraging the `helper` with Jack but it means project devs need to understand how that is structured internally

it needs to be in an overlay to make it into the `.my127ws` directory and then get copied onto the docker image?

Reply:
the project directory gets copied into /app though right? I suppose it might be too late in the step cycle tho

Reply:
yeah it does much later (after go mod download) to improve cacheability of the earlier layers